packages/packages/top-bar/src/components/ui/toolbar-menu-item.tsx

```diff
@@ -1,35 +1,21 @@
-import { IconButton, styled, svgIconClasses, IconButtonProps } from '@elementor/ui';
-import Tooltip from './tooltip';
-
-const Button = styled( IconButton )( () => ( {
-	borderRadius: '8px',
-	padding: '6px',
-	color: '#fff',
-	fontSize: '14px',
-	fontWeight: 300,
-	display: 'flex',
-	alignItems: 'center',
-
-	'&:hover, &:active': {
-		backgroundColor: 'rgba(255, 255, 255, 0.1)',
-	},
-
-	[ `& .${ svgIconClasses.root }` ]: {
-		fill: '#fff',
-		width: '20px',
-		height: '20px',
-	},
-} ) );
+import { IconButton, IconButtonProps, Tooltip, styled } from '@elementor/ui';
 
 type Props = IconButtonProps & {
 	title?: string;
 	selected?: boolean;
 }
 
+// The 'a' tag color is affected on hover by a global CSS color that applies on 'body a:hover {}'.
```

Comment:
Can we hardcoded it on the UI library?

Reply:
How will it work when other components try to inject into the top-bar?
I think at least it should lives another the theme level when you extends the theme that comes from the lib and override with code that specific for the "host".

Reply:
Done.